File: id2link.py
from clipboard import copy
while True:
    id_ = input("input netpad id,input \"q\" to quit:\n")
    if id_ != "q":
        print(f"https://www.netpad.net.cn/resource_web/course/#/{id_}\n")
        print(
            "<iframe width=\"772\" "
            "height=\"434\" "
            f"src =\"https://www.netpad.net.cn/thirdInnerPad.html?id={id_}#posts/{id_}\" " +
            "frameborder=\"0\" "
            "scrolling=\"auto\"> "
            "</iframe>\n")
    else:
        break
    num = input("input 1: copy first link, 2. second link\n")
    if num == "1":
        copy(f"https://www.netpad.net.cn/resource_web/course/#/{id_}")
    elif num == 2:
        copy("<iframe width=\"772\" "
             "height=\"434\" "
             f"src =\"https://www.netpad.net.cn/thirdInnerPad.html?id={id_}#posts/{id_}\" " +
             "frameborder=\"0\" "
             "scrolling=\"auto\"> "
             "</iframe>\n")
    print('\nlink copied to copyboard')

File: id2link.c
/*
* Copyright (c)2020
*All rights reserved.
* 文件名称：netpad.c
* 当前版本：1.0
* 作    者：wuyudi
* 完成日期：2020 年 07 月 27 日 星期一
*/
#include <stdio.h>
unsigned long id;
int main(void)
{
    printf("input netpad id,input \"Q\" to quit:\n");
    while (scanf("%lu", &id) && id != 'Q')
    {
        printf("https://www.netpad.net.cn/resource_web/course/#/%d\n", id);
        printf(
            "<iframe width=\"772\"\
            height=\"434\"\
            src =\"https://www.netpad.net.cn/thirdInnerPad.html?id=%d#posts/%d\"\
            frameborder=\"0\"\
            scrolling=\"auto\">\
            </iframe>\n",id,id);
        printf("input netpad id,input \"Q\" to quit:\n");
    }
    return 0;
}
